AirCasting/SearchAndFollow/AirBeamMeasurements/MeasurementsDownloaderResultModel.swift

```diff
@@ -0,0 +1,16 @@
+// Created by Lunar on 26/04/2022.
+//
+
+import Foundation
+
+struct MeasurementsDownloaderResultModel: Decodable {
+    let sensorName: String
+    let measurements: [theMeasurement]
+}
+
+struct theMeasurement: Decodable {
```

Comment:
capitalize the name plz

Reply:
Also - you named it `TheMeasurement` because `Measurement` was already taken. Why not putting this inside the ResultModel struct and just call it `Measurement`?

```Swift
struct MeasurementsDownloaderResultModel: Decodable {
     let sensorName: String
     let measurements: [Measurement]

    struct Measurement: Decodable { ... }
 }
```